In-house patch to remove unnecessary dependencies from Swift's
requirements files. The specific reasons are as follows:

greenlet	No longer applicable

--- swift-1.10.0/requirements.txt.orig	2013-10-17 07:07:33.000000000 -0700
+++ swift-1.10.0/requirements.txt	2014-05-24 23:17:47.564223065 -0700
@@ -1,7 +1,6 @@
 pbr>=0.5.21,<1.0
 dnspython>=1.9.4
 eventlet>=0.9.15
-greenlet>=0.3.1
 netifaces>=0.5
 pastedeploy>=1.3.3
 simplejson>=2.0.9

--- swift-1.10.0/swift.egg-info/requires.txt.orig	2013-10-17 07:08:06.000000000 -0700
+++ swift-1.10.0/swift.egg-info/requires.txt	2014-05-24 23:17:53.923411767 -0700
@@ -1,9 +1,8 @@
 pbr>=0.5.21,<1.0
 dnspython>=1.9.4
 eventlet>=0.9.15
-greenlet>=0.3.1
 netifaces>=0.5
 pastedeploy>=1.3.3
 simplejson>=2.0.9
 xattr>=0.4
-python-swiftclient
\ No newline at end of file
+python-swiftclient
